docs/fieldnorm/control_of_documents.md

```diff
@@ -1,25 +1,91 @@
-- Software development, and hence, corresponding documentation is prone to frequent updates. With the QMS of the laboratory, there are often waiting steps for different members of the laboratory to verify or authorize documentation. Hence, there might be a delay in release of a new version as the documentation of the previous release is not authorized yet.
-- add more
+_Bioinformaticians_ work in a fairly different environement than biologists, and the classic Quality Management System (QMS) offered to biologists are often impractical for _Bioinformaticians_ needs. Here is a list of the most wanted features :
 
-To allow for a git-like QMS to be used by bioinformaticians the following aspects need to be considered, and documented within the QMS how compliance is achieved. ISO-15189:2022, 8.3.1
+- Code blocks can be copy/pasted safely, as opposed to standard QMS with a word or pdf format, which introduce erroneous changes on some special characters (such as "-")
+- Linking versions of the _documentation_ with the corresponding release number of the documented tool
+- Let the _documentation_ update process be in the hands of _Bioinformaticians_ when changes in software must take place immediately, without getting in a stale review process when involving laboratory members
 
-1. documents are uniquely identified;
-2. documents are approved for adequacy before issue by authorized personnel who have the =expertise and competence to determine adequacy;
-3. documents are periodically reviewed and updated as necessary;
-4. relevant versions of applicable documents are available at points of use and, where necessary, their distribution is controlled;
-5. changes and the current revision status of documents are identified;
-6. documents are protected from unauthorized changes and any deletion or removal;
-7. documents are protected from unauthorized access;
-8. the unintended use of obsolete documents is prevented, and suitable identification is applied to them if they are retained for any purpose;
-9. at least one paper or electronic copy of each obsolete controlled document is retained for a specified time period or in accordance with applicable specified requirements.
+As specified in the ISO-15189, "Software procedures and related documentation needs to be controlled". This can be in any form or type of medium, as long as a certain set of requirements is achieved. Software are developped and distributed using a version control system, such as git or gitlab. Those often offer extra functionnalities related to code (e.g. : linking a new release with a set of solved issues), which make them good candidate to be a _Bioinformatician_ QMS system. In this document, we offer guidelines on how to comply with the norms while implementing a _bioinformatician_ friendly QMS. 
```

Comment:
The last "_bioinformatician"_ is not capitalized. We should also make sure that everyone uses the same capitalization rules for this so that this will be consistent throughout the whole fieldnorm.

Reply:
Software is (?) developed
functionalities <- not functionnalities
which makes them (?) 
comply to ISO-15189 using a _bioinformatician_ friendly QMS